.gitattributes

```diff
@@ -1,5 +1,6 @@
 * text=auto
 
+*.patch binary
```

Comment:
Ah, this is really annoying. Wish it wouldn't do this @conda-forge/core

Reply:
what would you prefer? without this, it becomes very annoying to control line endings.

you can use `dos2unix` and `unix2dos` to convert as needed based on the source.

Reply:
I would prefer if this line wasn't here. This makes it so that git won't show diffs for `.patch` files. This is really annoying as these files typically have to be updated manually (I just spend a bunch of time doing this in #75).

Anyway, can't line ending issues be handled by the `patch` program that applies the patch?

Alternatively, let each individual feedstock set this as appropriate.

Reply:
I use git diff —text locally to get around this limitation. 

In this particular merge request. I somewhat don’t even see any patch changed. 

Maybe open an issue on conda-smithy if you feel strongly to keep the conversation going.

Reply:
There were tons of patches I had to update in that PR. That's why I removed this line. 

BTW, if this really can't be made to work with the way conda build calls `patch` (which I'm sure it could), another simple solution would be to use a different extension for "binary" patches. The ".patch" extension is not required.

Reply:
We might be able to do either
```gitattributes
*.path -text
*.diff -text
```
or
```gitattributes
*.path binary diff
*.diff binary diff
```
. IDK which one would work/be preferred.

In any case, if this bothers you and you want to improve on the UX, you should open an issue for smithy, like @hmaarrfk suggested.

Reply:
You can do
```yml
skip_render:
  - .gitattributes
```
in `conda-forge.yml` and then the rerendering will not change it.

Reply:
That's even more argument to not have this here, then. The repos that actually need it can add it and use `skip_render`.

Reply:
https://github.com/conda-forge/conda-smithy/issues/1834